.travis.yml

```diff
@@ -3,9 +3,10 @@ language: python
 env:
     - PYCURL_SSL_LIBRARY=gnutls
 python:
-    - "3.6"
+    - "3.8"
```

Comment:
@lpramuk does your comment from https://github.com/SatelliteQE/robottelo/issues/7636#issuecomment-584662732 still apply? Can we bump robottelo minimum Python version to 3.8, or will entire CI break, since workers have Python 3.6 only?

Reply:
Automation runs in docker containers 
```
Running on zln--004k8x9ie78in on docker-openstack
```
currently providing python 3.7.5
```
platform linux -- Python 3.7.5, pytest-4.6.3, py-1.9.0, pluggy-0.13.1
```

Reply:
But upgrade automation is NOT !!!